changelogs/unreleased/uri-library-extension.md

```diff
@@ -0,0 +1,8 @@
+## feature/uri
+
+* Implemented new method `parse_many` in tarantool uri library,
+  which allowed to parse several URIs with different parameters
+  passed in different ways (as a table of string URIs and URIs
+  tables with URI and it's parameters).
+  Update `parse` method to make possible the same as for the new
+  `parse_many` method but only for single URI.
```

Comment:
You need to mention that the new many-parser is not compatible with the old one for URIs having `,` in them. These are valid URIs, but they won't be handled by this parser correctly.

Reply:
fixed